packages/notifier/src/types.js

```diff
@@ -48,7 +48,7 @@
  * update count, return the current record.
  * Otherwise, after the next state change, the promise will resolve to the
  * then-current value of the record.
- * @returns {Promise<UpdateRecord<T>>} resolves to the corresponding
+ * @returns {ERef<UpdateRecord<T>>} resolves to the corresponding
```

Comment:
Actually, I'm not convinced this is a good idea.  I remember when I explicitly stopped notifiers from sometimes returning non-promises, and that was a considerable ergonomic improvement.  Especially since using a notfier with `E(notifier).getUpdateSince(...)` always returns promises.

Reply:
Restored. Resolving.